File: web/src/model/victron_controller/dashboard/from_0_2_0_timers.ts
// @ts-nocheck
import {Timers as v0_2_0_Timers} from './v0_2_0/Timers'
import {Timers as dashboard_Timers} from './Timers'

export function convert__timers__from__0_2_0(from: v0_2_0_Timers): dashboard_Timers {
    return new dashboard_Timers (
        JSON.parse(JSON.stringify(from.entries))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_timers.rs


pub fn convert__timers__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::timers::Timers) -> crate::victron_controller::dashboard::timers::Timers {
    crate::victron_controller::dashboard::timers::Timers {
        entries: serde_json::from_value(serde_json::to_value(&from.entries).unwrap()).unwrap(),
    }
}